src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -1,26 +1,173 @@
+        const [ next, , ] = verifyButtons();
+        next.click();
+
+        screen.getByText('Destination');
+        screen.getByText('Secret access key');
+    });
+
+    // todo: Should we have the Back button here ?
+    // Maybe not b/c it doesn't lead anywhere
+
+    test('clicking Cancel loads landing page', () => {
+        const [ , , cancel ] = verifyButtons();
+        verifyCancelButton(cancel);
+    });
+});
+
+describe('Step Target environment', () => {
+    beforeEach(() => {
+        renderWithReduxRouter(<CreateImageWizard />);
+
+        // left sidebar navigation
+        const sidebar = screen.getByRole('navigation');
+        const anchor = getByText(sidebar, 'Target environment');
+
+        // load from sidebar
+        anchor.click();
+    });
+
+    test('clicking Next loads Registration', () => {
+        const [ next, , ] = verifyButtons();
+        next.click();
+
+        screen.getByText('Register the system');
+    });
+
+    test('clicking Back loads Release', () => {
+        const [ , back, ] = verifyButtons();
+        back.click();
+
+        // note: there doesn't seem to be anything better on this screen
+        screen.getByText('Red Hat Enterprise Linux (RHEL) 8.2');
+    });
+
+    test('clicking Cancel loads landing page', () => {
+        const [ , , cancel ] = verifyButtons();
+        verifyCancelButton(cancel);
+    });
+});
+
+describe('Step Registration', () => {
+    beforeEach(() => {
+        renderWithReduxRouter(<CreateImageWizard />);
+
+        // left sidebar navigation
+        const sidebar = screen.getByRole('navigation');
+        const anchor = getByText(sidebar, 'Registration');
+
+        // load from sidebar
+        anchor.click();
+    });
+
+    test('clicking Next loads Review', () => {
+        const [ next, , ] = verifyButtons();
+        next.click();
+
+        screen.getByText('Review the information and click Create image to create the image using the following criteria.');
+    });
+
+    test('clicking Back loads Target environment', () => {
+        const [ , back, ] = verifyButtons();
+        back.click();
+
+        screen.getByText('Destination');
+        screen.getByText('Secret access key');
+    });
+
+    test('clicking Cancel loads landing page', () => {
+        const [ , , cancel ] = verifyButtons();
+        verifyCancelButton(cancel);
+    });
+});
+
+describe('Step Review', () => {
+    beforeEach(() => {
+        renderWithReduxRouter(<CreateImageWizard />);
+
+        // left sidebar navigation
+        const sidebar = screen.getByRole('navigation');
+        const anchor = getByText(sidebar, 'Review');
+
+        // load from sidebar
+        anchor.click();
+    });
+
+    test('has 3 buttons', () => {
+        screen.getByRole('button', { name: /Create/ });
         screen.getByRole('button', { name: /Back/ });
         screen.getByRole('button', { name: /Cancel/ });
     });
+
+    // todo: add test for the Create button
```

Comment:
What is the Create button supposed to do? Maybe I have to mock the API to intercept the result from this action ?

Reply:
Yea, the create button sends the request and updates the store.

Reply:
I've decided to leave this for my next PR b/c ATM I have no idea how to do it and b/c it is different from everything else that is going on in this test for now.